### PR TITLE
Update README.md with warning

### PR DESCRIPTION
**Pull request by fedekau**
I ran out of words pretty shortly after starting. Is this enough so we can have public and have some peace of mind?

If this is not what you expected please give some direction and I will continue the works 😄 

See [rendered](https://github.com/tablexi/terraform_modules/blob/f577b5094f756d247d18d84658dc43642b573800/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
-# Terraform Modules
+# terraform_modules
+
+A group of terraform modules managed by Table XI (@tablexi). Most of this modules are things we do on every project or in a lot of them. This has been made open source so you can see some examples.
```
Comment (phoolish) [inline README.md]: Most of these modules

Comment (phoolish) [reply README.md]: ~~or in a lot of them~~

**Review by phoolish — APPROVED** [review]
2 suggestions, but otherwise looks good for a first pass.

**Comment by fedekau** [comment]
I will merge even if the tests are failing because there is no way that my changes to the README broke the tests.

**Comment by fedekau** [comment]
Does any of you know what caused the failure? It is failing on develop after the merge 😞

**Comment by pturley** [comment]
It looks like the latest AMIs have changed. We would need to regenerate the planshots with this. I will do that now in a new PR.

**Pull request merged**